px-data-table.html

```diff
@@ -342,6 +343,11 @@
       includeAllColumns: {
         type: Boolean,
         value: false
+      },
+
```

Comment:
Add comments for this property as with all other properties. Please be descriptive.
Could these values be hooked to variables that already exist?